src/tcgwars/logic/impl/gen4/Unleashed.groovy

```diff
@@ -1143,7 +1143,9 @@ public enum Unleashed implements LogicCardInfo {
             energyCost W
             onAttack {
               damage 10
-              damage 10, opp.bench.select()
+              if opp.bench {
```

Comment:
Could you wrap the `if` condition in parenthesis?  I believe it won't build otherwise

Reply:
Sorry, was having issues with git, will fix.